frontend/src/composables/useArticleXml.ts

```diff
@@ -16,27 +16,49 @@ import {
  * @param identifier A reference to the ELI/eId combination for which the article XML
  *  will be returned. Changing the value of the reference will load the XML for the
  *  new ELI/eId combination.
- * @returns A reference to the article XML or undefined if it is not available (or
- *  still loading).
  */
 export function useArticleXml(
   identifier: MaybeRefOrGetter<LawElementIdentifier | undefined>,
-): DeepReadonly<Ref<string | undefined>> {
+): {
+  /**
+   * A reference to the article XML or undefined if it is not available (or
+   *  still loading).
+   */
+  xml: DeepReadonly<Ref<string | undefined>>
+  /**
+   * A function to call so the xml is refreshed. It is not necessary to call this when the identifier changed.
+   */
+  refresh: () => Promise<unknown>
+} {
   const articleXml = ref<string>()
 
+  async function load(identifier: LawElementIdentifier) {
+    // TODO: Switch this to article XML once we have that. For now we'll display
+    // the entire amending law.
+    articleXml.value = await getAmendingLawXmlByEli(identifier.eli)
+  }
+
   watch(
     () => toValue(identifier),
     async (is, was) => {
       // Bail if only the object reference has changed, but the contents
       // are the same
       if (!is || (is.eli === was?.eli && is.eid === was?.eid)) return
 
-      // TODO: Switch this to article XML once we have that. For now we'll display
-      // the entire amending law.
-      articleXml.value = await getAmendingLawXmlByEli(is.eli)
+      await load(is)
     },
     { immediate: true },
   )
 
-  return readonly(articleXml)
+  async function refresh() {
```

Comment:
Todo: As discussed

- [ ] Move the update function from the component to the composable
- [ ] The API should return the updated data if the call is successful, then use that to update `articleXml`
- [ ] Feel free to remove `refresh` after that if you want